### PR TITLE
Tags filter not works if tags are uppercase

### PR DESCRIPTION
**Pull request by Se7en-IT**
When tags are uppercase, when i search on tags filter section, i don't find nothing.

### CODE REVIEW

File: bundles/AdminBundle/Controller/Admin/TagsController.php [hunk 1]
```diff
@@ -133,7 +133,7 @@ public function treeGetChildrenByIdAction(Request $request)
         if (!empty($request->get('filter'))) {
             $filterIds = [0];
             $filterTagList = new Tag\Listing();
-            $filterTagList->setCondition('`name` LIKE '. $filterTagList->quote('%'. $request->get('filter') .'%'));
+            $filterTagList->setCondition('LOWER(`name`) LIKE '. $filterTagList->quote('%'. $request->get('filter') .'%'));
```
Comment (brusch) [inline bundles/AdminBundle/Controller/Admin/TagsController.php:136]:
```suggestion
            $filterTagList->setCondition('LOWER(`name`) LIKE '. $filterTagList->quote('%'. strtolower($request->get('filter')) .'%'));
```

**Review by brusch — CHANGES_REQUESTED** [review]

**Comment by jdreesen** [comment]
Fixes #7606

**Comment by Se7en-IT** [comment]
I don't add strtolower because js file send it already lowercase.

Il ven 27 nov 2020, 13:26 Bernhard Rusch <notifications@github.com> ha
scritto:

**Comment by brusch** [comment]
Alright, thanks for you feedback!

**Comment by jdreesen** [comment]
I'd say: don't rely on the input and call `strtolower` nonetheless.

**Comment by Se7en-IT** [comment]
Ok perfect.

Il ven 27 nov 2020, 14:23 Jacob Dreesen <notifications@github.com> ha
scritto:

**Pull request merged**